Source/WebCore/html/HTMLPictureElement.cpp

```diff
@@ -75,5 +75,18 @@ bool HTMLPictureElement::isSystemPreviewImage()
 }
 #endif
 
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+bool HTMLPictureElement::isMultiRepresentationHEIC() const
+{
+    if (countChildNodes() != 2)
+        return false;
+
+    for (auto& element : childrenOfType<HTMLImageElement>(*this))
+        return element.isMultiRepresentationHEIC();
+
+    return false;
```

Comment:
The immediate return from the `for` loop was a bit unexpected, though it looks right. I wonder if this might be clearer?

```suggestion
    RefPtr firstImage = childrenOfType<HTMLImageElement>(*this).first();
    return firstImage && firstImage->isMultiRepresentationHEIC();
```